app/pages/settings/email.jsx

```diff
@@ -0,0 +1,297 @@
+import React from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import talkClient from 'panoptes-client/lib/talk-client';
+import AutoSave from '../../components/auto-save';
+import handleInputChange from '../../lib/handle-input-change';
+
+class EmailSettingsPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      page: 1,
+      projects: [],
+      projectPreferences: [],
+      talkPreferences: []
+    };
+    this.handleProjectPreferenceChange = this.handleProjectPreferenceChange.bind(this);
+    this.handleTalkPreferenceChange = this.handleTalkPreferenceChange.bind(this);
+    this.talkPreferenceOption = this.talkPreferenceOption.bind(this);
+    this.getProjectForPreferences(props.user);
+    this.getTalkPreferences();
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (!(this.state.page === prevState.page)) {
```

Comment:
Better to use `!==` here.

Reply:
Sure, I can see it's more readable. Are there any other reasons?

Reply:
Because you're checking whether two variables are not equal, and `!==` is the strict not equal comparison operator in JavaScript. 

Here you're testing if two values are strictly equal, then performing a second test to invert the first test.